openmls/src/group/core_group/new_from_external_init.rs

```diff
@@ -31,9 +31,8 @@ impl CoreGroup {
         tree_option: Option<&[Option<Node>]>,
         verifiable_public_group_state: VerifiablePublicGroupState,
     ) -> Result<ExternalCommitResult, ExternalCommitError> {
-        let ciphersuite = Config::ciphersuite(verifiable_public_group_state.ciphersuite())
-            .map_err(|_| ExternalCommitError::UnsupportedCiphersuite)?;
-        if !Config::supported_versions().contains(&verifiable_public_group_state.version()) {
+        let ciphersuite = verifiable_public_group_state.ciphersuite();
+        if verifiable_public_group_state.version() != ProtocolVersion::Mls10 {
```

Comment:
Does it make sense to even do the check here? If I understand correctly, the check is meant to determine if we support the version in the implementation in general rather than this being configurable in some way. But isn't the fact that we were able to deserialize the `verifiable_public_group_state` in the first place an indicator that we do support it? Probably not something we want to solve here, but if my understanding is correct, we might want to file this as a follow-up. 

(The same goes for other places where we check some variable against `ProtocolVersion::Mls10`.)

Reply:
We have one other version we support in the enum but don't actually support in the code. So keeping the check makes sense I think. And it's a canary when starting to work on actually adding support for other versions.

Reply:
Then we should probably remove that version from the enum. But that's nothing we need to do in this PR.